cli/tests/error_005_missing_dynamic_import.ts.out

```diff
@@ -1,2 +1 @@
-error: Cannot resolve module "[WILDCARD]/bad-module.ts" from "[WILDCARD]/error_005_missing_dynamic_import.ts".
-    at file:///[WILDCARD]/error_005_missing_dynamic_import.ts:3:26
+error: Uncaught (in promise) TypeError: Cannot resolve module "[WILDCARD]/cli/tests/bad-module.ts".
```

Comment:
Is is possible to keep `at <location>`?

Reply:
Maybe, I will look into it... of course if the promise is caught, it should have an error stack. I wonder why we don't print error stacks for uncaught promises... 🤔

Reply:
That's a great question 🤔  it's really strange, @nayeemrmn any ideas?

Reply:
confirmed it is there, logging out a caught error:

```
TypeError: Cannot resolve module "file:///Users/kitsonk/github/deno/cli/tests/dynamic_import/bad.mjs" from "file:///Users/kitsonk/github/deno/cli/tests/dynamic_import/b.js".
    at file:///Users/kitsonk/github/deno/cli/tests/dynamic_import/b.js:1:0
```

I opened #8769.